rds-deployment-existing-ad-customrdsh/README.md

```diff
@@ -0,0 +1,16 @@
+# Create Remote Desktop Sesson Collection deployment
```

Comment:
Name needs to reflect custom image for RD session host

Also a better folder name would be rds-deployment-custom-image